src/platform/site-wide/sass/style.scss

```diff
@@ -1,10 +1,10 @@
 /*
- * NOTE: there is a version of this file for brand consolidation at
- * brand-consolidate.scss. If you make a change to this file, please
+ * NOTE: there is a version of this file for the regular vets.gov build at
```

Comment:
Is this comment still needed? We still have a vets.gov build?

Reply:
Seems it can be deleted, because it's referring to the `style.scss` inside of itself. Also, there will no longer be another file.